app/views/layouts/_header.html.erb

```diff
@@ -20,7 +20,7 @@
     <div class="collapse navbar-collapse" id="header-navbar-collapse">
       <form class="navbar-form navbar-left" id="searchform" autocomplete="off">
         <div class="input-group">
-          <input type="text" id="searchform_input" style="min-width: 220px;" class="form-control search-query typeahead" role="search" qryType="all" placeholder="<%= t('layout._header.search') %>"  value="<%= params[:query] %>" required>
+          <input type="text" id="searchform_input" style="min-width: 220px;" minlength="3" oninput="checkInputLength(this)" class="form-control search-query typeahead" role="search" qryType="all" placeholder="<%= t('layout._header.search') %>"  value="<%= params[:query] %>" required>
```

Comment:
Hi, i'm sorry to circle back here, but I've been thinking hard about this and I think the ideal may really be to not show an error message, but to simply not send the request until there are at least 3 characters. Does `minlength` provide this, or would we need to re-think how this is implemented a bit? 

Can you confirm if the AJAX requests are being sent as you type 3 characters, or if this is displaying an alert but not limiting the requests actually sent? 

Thanks for your help with this! I'm sorry we've modified our goals with this a bit, but I really want the search experience to be smooth for people. I appreciate your patience!